marketplace/server/datasources.json

```diff
@@ -3,5 +3,15 @@
     "name": "db",
     "connector": "memory",
     "file": "db.json"
+  },
+  "ComposeMongo": {
+    "host": "",
+    "port": 0,
+    "url": "mongodb://productAdmin:loopback123@aws-us-east-1-portal.26.dblayer.com:17243,aws-us-east-1-portal.24.dblayer.com:17243/marketplaceDemo?ssl=true",
```

Comment:
For this, no. I spun this up deliberately for demo purposes. It's a paid trial but Ashley is trying to hook us up to make this paid for, in exchange for us saying we're using Compose.